lambda_funcs/submission-created.js

```diff
@@ -0,0 +1,39 @@
+// NOTE: This is to test on deploy link. When approved I can change this to our proper URL
+const URL = `ideas42ventures`
+const headers = {
+  "Access-Control-Allow-Origin": `${URL}`,
+  "Access-Control-Allow-Headers": "Content-Type",
+  "Access-Control-Allow-Methods": "POST",
+};
+
+exports.handler = function (event, context, callback) {
+
+  verifyReq(callback,event)
+
+  const email = JSON.parse(event.body).payload.email;
+  const isEmail = /^[^@]+@[^@]+\.[^@]+$/.test(email);
+
+  if (!isEmail) {
+    return respondWith(callback,400,"This is an invalid email")
+  }
+  return respondWith(callback,200,"Thanks for signing up")
+};
+
+function verifyReq(cb,event) {
```

Comment:
Can we get prettier and eslint to run on all js files in this directory, both in the `prettier` script and in the `lint-staged` script. Right now, we only run those on files within `src`. Can either move the directory, or update the scripts, whatever works best.

Reply:
Moved this file into the `src` dir.